Makefile

```diff
@@ -116,7 +116,7 @@ uninstall: ## Remove all rukpak resources from the cluster
 ##################
 # Build and Load #
 ##################
-.PHONY: build plain unpack core build-container kind-load kind-load-bundles kind-cluster
+.PHONY: build plain unpack core build-container kind-load kind-load-bundles kind-cluster kind
```

Comment:
This looks like a duplicate .PHONY target as we also have one in the hack/tools section

Reply:
Added for redundancy but will remove.